packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dedicatedCloud.service.js

```diff
@@ -345,6 +345,15 @@ class DedicatedCloudService {
       .then(({ data }) => data.filter((item) => item !== null));
   }
 
+  getDatacenterNsxtOption(serviceName) {
```

Comment:
```suggestion
  getDatacenterNsxtEnabled(serviceName) {
```

Because we don't render option but only object with `enabled`attribute ?

Reply:
Fixed!